_posts/2020-07-10-removing-negative-words.markdown

```diff
@@ -0,0 +1,123 @@
+
+Moving away from the "master / slave" terminology, however, has seen more controversy. As such, the following sections will be dedicated to that particular change.
+
+As Matthew Ahrens explains in his [pull request](https://github.com/openzfs/zfs/pull/10435) removing all possible references to the term "slave" from OpenZFS' `zfs` repository:
+
+> The horrible effects of human slavery continue to impact society. The casual use of the term "slave" in computer software is an unnecessary reference to a painful human experience.
+
+Empathy in itself is a pretty good motivation for change and knowing that these terms are hurtful and can easily be replaced is sufficient to prompt us to act. However, we also believe in understanding why we are doing things and how what we are doing fits with our company culture and purpose. Simply following the trend without questioning why it's important, how it's beneficial and where it fits in a bigger picture context isn't enough.
+
+Understanding these points is paramount to create a culture of ambassadors of change. Questioning, done with an open mind and actual desire to understand, is healthy and desirable.
+
+### Why is a change in language needed?
+
+There is an intrinsic relationship between language and power. And there is an unstoppable evolution in language. Every day, new terms arise and old terms become obsolete.
+
+The "master / slave" metaphore is clearly connected to slavery. The metaphore adopted in technology isn't master / copy, or master / student. The term master has several different meanings and it isn't, in itself, problematic. However, when used together with slave, in a metaphore referring to the "master / slave" relationship, it becomes a problem.
+
+The term "master" as used in git is also not used in the context of mastery of knowledge or even the original copy. As Bastien Nocera points out in his [email in reponse to the proposal to remove the "master" name from GNOME repositories](Bastien Nocera), the term "master" first appeared in git in a [CVS helper script](https://github.com/git/git/commit/3e91311ae750af9bf2e3517b1e701288ac3066b9) and was likely chosen because [BitKeeper uses "master" for its main branch](http://www.bitkeeper.org/tips.html#_how_do_i_rebase_my_work_on_top_of_a_different_changeset). According to BitKeeper's documentation, the term is a reference to the master / slave relationship, [as they refer to master / slave repositories](https://github.com/bitkeeper-scm/bitkeeper/blob/master/doc/HOWTO.ask#L223) and, in BitKeeper, [repositories and branches are the same thing](https://users.bitkeeper.org/t/branching-with-bk/158/2).
```

Comment:
`[email in reponse to the proposal to remove the "master" name from GNOME repositories](Bastien Nocera)`

@ABizzinotto you're missing the link there. Also there's a typo in "response"